stanford_soe_r25.install

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * @file
+ */
+
```

Comment:
use `hook_schema` to create a table to hold all this data instead of a variable.